### PR TITLE
Cthl 1550 new mutators

### PR DESCRIPTION
**Pull request by vasiliyyudin**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Markus Zimmermann
+Copyright (c) 2021 AvitoTech
```
Comment (ShamanR) [inline LICENSE:3]: Ммм.....это вообще законно?

**Review by Enapiuz — APPROVED** [review]
Everything is ok, but a question about a license is still opened.

File: mutator/loop/condition.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	}
+
+	condition, ok := n.Cond.(*ast.BinaryExpr)
+	if !ok {
+		return nil
+	}
+
+	originalX := condition.X
+	originalOp := condition.Op
+	originalY := condition.Y
```
Comment (ShamanR) [inline mutator/loop/condition.go:29]: is OpPos important?

File: mutator/loop/range_break.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+// MutatorLoopRangeBreak implements a mutator to add a break to range-loop body.
+func MutatorLoopRangeBreak(_ *types.Package, _ *types.Info, node ast.Node) []mutator.Mutation {
+	var list []ast.Stmt
+
+	switch n := node.(type) {
+	case *ast.BlockStmt:
+		list = n.List
+	case *ast.CaseClause:
+		list = n.Body
+	}
```
Comment (ShamanR) [inline mutator/loop/range_break.go:33]: if we will add deafult case we can make return asap

File: mutator/loop/range_break.go [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+// MutatorLoopRangeBreak implements a mutator to add a break to range-loop body.
+func MutatorLoopRangeBreak(_ *types.Package, _ *types.Info, node ast.Node) []mutator.Mutation {
+	var list []ast.Stmt
+
+	switch n := node.(type) {
+	case *ast.BlockStmt:
+		list = n.List
+	case *ast.CaseClause:
+		list = n.Body
```
Comment (ShamanR) [inline mutator/loop/range_break.go:32]: can u kindly explain - `CaseClause` how we can get a RangeStmt inside?
may be some code examples ?

**Review by AlexanderAsmakov — APPROVED** [review]

File: mutator/loop/condition.go (hunk 2 above)
Comment (vasiliyyudin) [reply mutator/loop/condition.go:29]: nope for new nodes

File: LICENSE (hunk 1 above)
Comment (vasiliyyudin) [reply LICENSE]: fixed

File: mutator/loop/range_break.go (hunk 4 above)
Comment (vasiliyyudin) [reply mutator/loop/range_break.go:32]: any range-based for inside case block
	
	case "darwin":
		var cow = []float64{1.0, 2.0, 3.0}
		for _, v := range cow {
			fmt.Print(v)
		}

File: mutator/loop/range_break.go (hunk 3 above)
Comment (vasiliyyudin) [reply mutator/loop/range_break.go:35]: ty, done

**Pull request merged**